browser/ui/views/tabs/brave_tab_style_views.inc.cc

```diff
@@ -113,6 +104,26 @@ SkPath BraveVerticalTabStyle::GetPath(
   }
 #endif
 
+  if (!ShouldShowVerticalTabs()) {
+    // Horizontal tabs should have a visual gap between them, even if their view
+    // bounds are touching or slightly overlapping. Create a visual gap by
+    // insetting the bounds of the tab by the required gap plus overlap before
+    // drawing the rectangle.
+    aligned_bounds.Inset(
+        gfx::InsetsF::VH(brave_tabs::kHorizontalTabVerticalSpacing * scale,
+                         brave_tabs::kHorizontalTabInset * scale));
+
+    // For maximized and fullscreen windows, extend the tab hit test to the top
+    // of the tab, encompassing the top spacing. This makes it easy to click on
+    // tabs by moving the mouse to the top of the screen.
+    if (path_type == TabStyle::PathType::kHitTest &&
+        (tab()->GetWidget()->IsMaximized() ||
+         tab()->GetWidget()->IsFullscreen())) {
+      aligned_bounds.Inset(gfx::InsetsF::TLBR(
```

Comment:
optional: There's gfx::Rect::Outset() and would be perfect for this, but I'm okay with the current version 👍

Reply:
Done